tests/client/features/OffreEmploi/Rechercher/RechercherOffreEmploi.test.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * @jest-environment jsdom
+ */
+import '@testing-library/jest-dom';
+
+import { fireEvent, render, screen, waitFor, within } from '@testing-library/react';
+import { mockLargeScreen, mockSmallScreen } from '@tests/client/window.mock';
+import { anOffreEmploiService } from '@tests/fixtures/client/services/offreEmploiService.fixture';
+
+import { DependenciesProvider } from '~/client/context/dependenciesContainer.context';
+import RechercherOffreEmploiPage from '~/pages/emplois';
+
+describe('RechercherOffreEmploi', () => {
+  beforeAll(() => {
+    mockSmallScreen();
+  });
+
+  describe('quand on arrive sur la page', () => {
+    it('affiche un formulaire pour la recherche d\'offres d\'emploi et aucun résultat', () => {
+      const offreEmploiServiceMock = anOffreEmploiService();
+
+      render(
+        <DependenciesProvider offreEmploiService={offreEmploiServiceMock}>
+          <RechercherOffreEmploiPage/>
+        </DependenciesProvider>,
+      );
+
+      const résultatRechercheOffreEmploiList = screen.queryAllByTestId('RésultatRechercheOffreEmploi');
+      const rechercheOffreEmploiNombreRésultats = screen.queryByTestId('RechercheOffreEmploiNombreRésultats');
+
+      expect(résultatRechercheOffreEmploiList).toHaveLength(0);
+      expect(rechercheOffreEmploiNombreRésultats).not.toBeInTheDocument();
+    });
+  });
+
+  describe('quand la recherche est lancée', () => {
+    it('affiche les résultats de recherche et le nombre de résultats', async () => {
+      const offreEmploiServiceMock = anOffreEmploiService();
+
+      render(
+        <DependenciesProvider offreEmploiService={offreEmploiServiceMock}>
+          <RechercherOffreEmploiPage/>
+        </DependenciesProvider>,
+      );
+
+      const containerRechercheMotClé = screen.getByTestId('InputRechercheMotClé');
+      const inputRechercheMotClé = within(containerRechercheMotClé).getByRole('textbox');
+      fireEvent.change(inputRechercheMotClé, { target: { value: 'boulanger' } });
+      const buttonRechercher = screen.getByTestId('ButtonRechercher');
+      fireEvent.click(buttonRechercher);
+      const résultatRechercheOffreEmploiList = await screen.findAllByTestId('RésultatRechercherOffreEmploi');
+      const rechercheOffreEmploiNombreRésultats = await screen.findByTestId('RechercheOffreEmploiNombreRésultats');
+
+      expect(résultatRechercheOffreEmploiList).toHaveLength(3);
+      expect(rechercheOffreEmploiNombreRésultats).toHaveTextContent('3 offres d\'emplois');
+      expect(offreEmploiServiceMock.rechercherOffreEmploi).toHaveBeenCalledWith({
+        motCle: 'boulanger',
+        typeDeContrats: '',
+      });
+    });
+  });
+
+  describe('quand les filtres avancés sont ouverts', () => {
```

Comment:
```suggestion
  describe('quand les filtres avancés sont ouverts en mode mobile', () => {
```
Besoin de préciser `en mode mobile` ou pas ?